### ISSUE TITLE
Crashing on aiReleaseImport

### ISSUE THREAD

**Issue by markodevv** [assimp/assimp#3750]
**Describe the bug**
When trying to release scene resources, program crashes on aiReleaseImport(). 
This happens only with .fbx files, have tested the .obj files and it works fine.

```
    const aiScene* scene = aiImportFile(name, aiProcess_CalcTangentSpace |
                                              aiProcess_Triangulate      |
                                              aiProcess_SortByPType);
    if (!scene)
    {
        DEBUG_PRINT("%s", aiGetErrorString());
        ASSERT(0);
    }

    ....

    aiReleaseImport(scene); // Crashes here
```
Here is the file: [Mesh.zip](https://github.com/assimp/assimp/files/6342437/Mesh.zip)

**Desktop**
 - OS: Windows 10
 - Assimp: 5.0.1

**Comment by kimkulling** [comment]
At first thanks a lot for your bug report. I am wondering why the tests for the FBX-Importer haven't found those crashes. Do you have any kind of small example to reproduce the crash? This would help us a lot.

Kim

**Comment by bisevac-marko** [comment]
@kimkulling Yeah i uploaded it to filebin because github won't let me. It's this file [https://filebin.net/ck87d2bvahf1pvt9](url). I have lodaded other fbx files and it works fine, it just seems to crash on this particular one.

**Comment by bisevac-marko** [comment]
@kimkulling Just loading and right away and releasing this FBX file srashes.

**Comment by kimkulling** [comment]
The problem is caused by a misleading bone loading process. The release call at the end is just because the scene is a nullptr.

**Comment by tellypresence** [comment]
Closing as author never provided 3D model for investigation as requested
